lib/jekyll/commands/serve.rb

```diff
@@ -1,142 +1,192 @@
-# -*- encoding: utf-8 -*-
 module Jekyll
   module Commands
     class Serve < Command
-
       class << self
+        COMMAND_OPTIONS = {
+          "detact" => ["-B", "--detach", "Run the server in the background (detach)"],
```

Comment:
That's been fixed locally I should probably push up the new changes with the new handler.